tests/unit/fem/test_build_dof_to_arrays.cpp

```diff
@@ -464,7 +531,8 @@ Mesh * GetMesh(MeshType type)
          mesh->AddTet(8, 2, 7, 5);
          break;
    }
-   mesh->FinalizeTopology();
+   mesh->FinalizeTopology(true);
+   for (int i = 0; i < mesh->GetNBE(); i++) { mesh->SetBdrAttribute(i, 1); }
```

Comment:
```suggestion
   mesh->FinalizeTopology();
```
This could be reverted as `Point` has been fixed, but it is not very important 😉